.github/workflows/buildsCI.yaml

```diff
@@ -427,7 +445,7 @@ jobs:
         sudo apt-get install -qq libusb-1.0-0-dev;
         sudo apt-get install -qq libgtk-3-dev;
         sudo apt-get install libglfw3-dev libglfw3;
-        python3 -m pip install numpy
+        sudo apt-get install python3-numpy
```

Comment:
If we want to use `pip` then, we should use `venv` in U24. Instead we can use `apt-get install` itself.